zerver/migrations/0122_disallow_markdown_chars_in_stream_and_user_name.py

```diff
@@ -2,6 +2,28 @@
 from django.db import models, migrations
 from django.db.backends.postgresql_psycopg2.schema import DatabaseSchemaEditor
 from django.db.migrations.state import StateApps
+from typing import Text
+
+def stream_name_in_use(apps, stream_name, realm_id):
```

Comment:
ah, I see you put the duplicate stream code in here :). 
One thing to note about migrations is that you basically never want to split them over multiple commits, or modify a past migration. Once a migration is run by an installation, that installation will never run the code in that migration again, so changing it after the fact doesn't have much of an effect. (Does that make sense?)